local/path/envoy.yaml

```diff
@@ -0,0 +1,33 @@
+                # Routing configuration for incoming requests.
```

Comment:
I don't fully understand why we've added this file in this PR?

Reply:
It wasn't meant to be committed; was just there from my local development (there was nothing sensitive in it). I've removed.